.github/workflows/coverage.yaml

```diff
@@ -2,40 +2,50 @@ name: Code Coverage
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 env:
   CARGO_INCREMENTAL: 0
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: 1
 
-defaults:
-  run:
-    shell: bash -ieo pipefail {0}
-
 jobs:
   coverage:
     runs-on: ubuntu-latest
-    container:
-      image: docker://openrr/openrr-coverage-ci:ros2
-      options: --security-opt seccomp=unconfined
-    env:
-      HOME: /root
     steps:
       - uses: actions/checkout@v2
-      - run: ci/ubuntu-install-dependencies.sh
+      - run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+            cmake build-essential libudev-dev xorg-dev libglu1-mesa-dev libasound2-dev libclang-dev \
```

Comment:
Is there any reason why avoid using `ci/ubuntu-install-dependencies.sh`?

Reply:
And it might be better to be updated.

```
cmake build-essential (openrr-planner (assimp-sys))
libudev-dev (arci-gamepad-gilrs)
xorg-dev libglu1-mesa-dev (openrr-gui (iced))
libasound2-dev (arci-speak-audio)
```

Reply:
Good point. That was a change made when I was trying with the ros's docker image, so I think we can probably use `ci/ubuntu-install-dependencies.sh`.